manim/mobject/geometry.py

```diff
@@ -710,6 +710,18 @@ def set_opacity(self, opacity, family=True):
 
 
 class DashedLine(Line):
+    """A dashed Line : - - - - - -
```

Comment:
Suggestion (non-blocking): I'm not sure the visual aid here is necessary :sweat_smile: 
```suggestion
    """A dashed Line.
```

Reply:
Ok, I just thought this would be an easter egg, but I don't mind removing it.